contingency/contingency-dsl/src/test/java/com/powsybl/contingency/dsl/GroovyDslContingenciesProviderTest.java

```diff
@@ -179,4 +179,20 @@ public void withComparison() throws IOException {
                 .getContingencies(network);
         assertTrue(contingencies.isEmpty());
     }
+
+    @Test
+    public void testExtension() throws IOException {
+        writeToDslFile("contingency('test') {",
+                "        equipments 'NHV1_NHV2_1'",
+                "           probability {",
+                "              base 0.1",
+                "              tsName 'myTs'",
+                "           }",
```

Comment:
It just a detail, but:
- could you indent the code propertly
- could you add another contingency and check below that there is no extension for the second contingency